save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/HtmlElements.kt

```diff
@@ -21,13 +24,58 @@ import kotlinx.js.jso
 /**
  * @param project
  */
-fun ChildrenBuilder.privacySpan(project: Project) {
-    span {
-        className = ClassName("border ml-2 pr-1 pl-1 text-xs text-muted ")
-        style = jso {
-            borderRadius = "2em".unsafeCast<BorderRadius>()
+fun ChildrenBuilder.statusSpan(project: Project) {
+    when (project.status) {
+        ProjectStatus.CREATED -> span {
+            className = ClassName("border ml-2 pr-1 pl-1 text-xs text-muted ")
+            style = jso {
+                borderRadius = "2em".unsafeCast<BorderRadius>()
+            }
+            +if (project.public) "public" else "private"
+        }
+        ProjectStatus.DELETED -> span {
+            className = ClassName("border ml-2 pr-1 pl-1 text-xs text-muted ")
+            style = jso {
+                borderRadius = "2em".unsafeCast<BorderRadius>()
+            }
+            +"deleted"
+        }
+        ProjectStatus.BANNED -> span {
+            className = ClassName("border ml-2 pr-1 pl-1 text-xs text-danger ")
+            style = jso {
+                borderRadius = "2em".unsafeCast<BorderRadius>()
+            }
+            +"banned"
+        }
+    }
+}
+
+/**
+ * @param organization
+ */
+fun ChildrenBuilder.statusSpan(organization: Organization) {
```

Comment:
`Organization` is not available on FE, please use `OrganizationDto`

Reply:
Done